json_theme/lib/src/codec/theme_encoder.dart

```diff
@@ -2057,7 +2057,7 @@ class ThemeEncoder {
   ///  * [encodeColor]
   ///  * [encodeShapeBorder]
   ///  * [encodeTextStyle]
-  static Map<String, dynamic>? encodeDialogTheme(DialogTheme? value) {
+  static Map<String, dynamic>? encodeDialogTheme(DialogThemeData? value) {
```

Comment:
This should be named encodeDialogThemeData since it now takes a DialogThemeData.

Also, curious why there isn't a matching ThemeDecoder change.